k-distribution/tests/regression-new/llvm-krun/Makefile

```diff
@@ -0,0 +1,34 @@
+DEF=test
+EXT=
+TESTDIR=.
+KOMPILE_BACKEND=llvm
+KOMPILE_FLAGS+=--gen-glr-bison-parser
+
+CHECK=| diff - test.out
+LLVM_KRUN_FLAGS=-p -d ./test-kompiled
```

Comment:
This could of course all be abstracted (cf. the existing `krun` tests), but I don't think it's worth doing for the sake of this single test case.